p2p/connection/recv_buffer.go

```diff
@@ -1,9 +1,12 @@
 package connection
 
+import "github.com/thetatoken/ukulele/common"
+
 type RecvBuffer struct {
 	workspace []byte
 
-	config RecvBufferConfig
+	config  RecvBufferConfig
+	chanSeq map[common.ChannelIDEnum]uint
```

Comment:
RecvBuffer is actually attached to the Channel object, so may not need a map here? See channel.go